### ISSUE TITLE
Leveling up is broken

### ISSUE THREAD

**Issue by AVBL** [KiwifruitDev/ArkhamRevivedSelfHosted#1]
1. After starting the game with the ArkhamReviving server, all factions already have level 25.
2. After reaching level 26, then the level does not grow at all. Because of this, it is impossible to get some achievements related to getting level 30.

**Comment by Azrial-The-Saint** [comment]
yeah same issue here. This is an amazing mode, it would be great if we could get the leveling system to work as well.
there is a growing community all using this great tool.
